README.FreeBSD

```diff
@@ -16,18 +16,18 @@ git submodule update --init --recursive
 cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo
 gmake
 
-3. Set up python 2.7
+3. Set up python 3.x
 -------------
 
 Install testing dependences either from packages or from pip.
 
 3.1. From packages:
 -------------
-pkg install python27 py27-yaml py27-daemon py27-msgpack
+pkg install python37 py37-yaml py37-gevent py37-six py37-msgpack
```

Comment:
python-msgpack is shipped with test-run as a submodule.

Reply:
Done

```
--- a/README.FreeBSD
+++ b/README.FreeBSD
@@ -23,7 +23,7 @@ Install testing dependences either from packages or from pip.
 
 3.1. From packages:
 -------------
-pkg install python37 py37-yaml py37-gevent py37-six py37-msgpack
+pkg install python37 py37-yaml py37-gevent py37-six
 
 3.2. From pip:
 -------------
```